x/ibc/applications/transfer/types/keys.go

```diff
@@ -25,6 +25,9 @@ const (
 
 	// QuerierRoute is the querier route for IBC transfer
 	QuerierRoute = ModuleName
+
+	// DenomPrefix is the prefix used for internal SDK coin representation.
+	DenomPrefix = "ibc"
```

Comment:
should this be a var? I think the difference would be whether we want to allow an application creator to choose their own prefix

Reply:
yeah, I think it makes sense?

Reply:
Actually, lets just leave as a const, if it is a var another module could simply change the value of the prefix which we can't allow for safety